exercice 6 slicing numpy.py

```diff
@@ -0,0 +1,17 @@
+from scipy import misc 
+import matplotlib.pyplot as plt 
+face = misc.face(gray=True)
+
+def grey_print(file) :
+    plt.imshow(file, cmap=plt.cm.gray)
+    plt.show()
+
+
+def zoom(image) :
+    (axe1, axe2) = image.shape
+    image_Zoom = image[int(0.25*axe1):int(0.75*axe1),int(0.25*axe2):int(0.75*axe2)]
+    image_Zoom[image_Zoom > 225] = 255 #augmenter le contraste 
```

Comment:
target all the value of the matrix imageZoom > 225 ?

Reply:
it's a method called 'boolean indexing' the guy on the video uses the term 'mask' to describe it. This part of the program make the picture "whiter"